pkg/pool-weighted/contracts/WeightedPoolProtocolFees.sol

```diff
@@ -64,23 +64,43 @@ abstract contract WeightedPoolProtocolFees is BaseWeightedPool, ProtocolFeeCache
             );
     }
 
-    function _afterJoinExit(
+    function _getJoinExitProtocolFees(
         bool isJoin,
```

Comment:
Why don't need this any more, since we have both post and pre supply

Reply:
@EndymionJkb I think Nico means here that you could replace the usages of `isJoin` with `postJoinExitSupply > preJoinExitSupply`

Reply:
Ah - that makes sense (now addressed with #1652 ). It looked like the *function* wasn't needed anymore somehow